tests/data/2011_february_aa_flight_paths.csv

```diff
@@ -0,0 +1,179 @@
+start_lat,start_lon,end_lat,end_lon,airline,airport1,airport2,cnt
```

Comment:
I'm wondering why we don't pull the data from the original website but copy the file.

Reply:
It's simpler. Is there a license concern?